Ports/pacman/package.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=pacman
+version=git
+workdir=pacman-master
+files="https://github.com/ebuc99/pacman/archive/refs/heads/master.tar.gz pacman-master.tar.gz 3016af69e168e0dd87a079bbfb35d995cf2e7bc1b710716c81ac99323bb4f7d5"
+auth_type=sha256
+useconfigure=true
+depends=("SDL2" "SDL2_image" "SDL2_ttf" "SDL2_mixer")
```

Comment:
Should be one line each.

```suggestion
depends=(
    "SDL2"
    "SDL2_image"
    "SDL2_mixer"
    "SDL2_ttf"
)
```